src/registrar/assets/sass/_theme/_typography.scss

```diff
@@ -27,6 +27,10 @@ h2 {
 .usa-form,
 .usa-form fieldset {
   font-size: 1rem;
+  legend em {
```

Comment:
Now that our field headings and subheadings are included as legend items in the screenreader fix, we're setting font-size to 1rem to preserve their font size 16px.